src/Http/ErrorResponseBuilder.php

```diff
@@ -85,11 +85,11 @@ public function addData(array $data):ErrorResponseBuilder
     /**
      * Set the error code and optionally an error message.
      *
-     * @param  string|null       $errorCode
+     * @param  mixed|null       $errorCode
      * @param  string|array|null $message
      * @return self
      */
-    public function setError(string $errorCode = null, $message = null):ErrorResponseBuilder
+    public function setError($errorCode = null, $message = null):ErrorResponseBuilder
```

Comment:
It slipped in actually. I was going to move it into a separate PR... but since it is here I had a use case where the errorCode was actually an integer value. 

Example errors.php
`<?php

use App\Enums\ErrorCode;

return [

    /*
    |--------------------------------------------------------------------------
    | Error Message Language Lines
    |--------------------------------------------------------------------------
    |
    | The following language lines are used by the Laravel Responder package.
    | When it generates error responses, it will search the messages array
    | below for any key matching the given error code for the response.
    |
    */
    ErrorCode::UNKNOWN_ERROR => 'Unknown Issue.',

];`